packages/main/src/RangeSlider.js

```diff
@@ -99,6 +98,10 @@ class RangeSlider extends SliderBase {
 		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
 	}
 
+	onEnterDOM() {
+		ResizeHandler.register(this, this._resizeHandler);
```

Comment:
I think it was fine having this in the base class. Why did you move it here? I would suggest you return it to SliderBase and call `super.onEnterDOM` anywhere you want to have something move in this method.

Reply:
Hello Vladi, thank you for the review. I moved the onEnterDOM() from both Slider and RangeSlider components in the SliderBase parent class, as you suggested, but the RangeSlider change is made in the RangeSlider pull request.